OrbitUIProtos/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(OrbitUIProtos)
```

Comment:
As stated in the other comment, maybe "OrbitClientProtos"?

Reply:
Renamed it, thanks!